crates/ruff/src/rule_redirects.rs

```diff
@@ -18,9 +18,8 @@ static REDIRECTS: Lazy<HashMap<&'static str, &'static str>> = Lazy::new(|| {
         // The following are here because we don't yet have the many-to-one mapping enabled.
         ("SIM111", "SIM110"),
         // The following are deprecated.
-        ("C", "C4"),
```

Comment:
No longer need to redirect these at all, since they have their own custom selectors.